tests/cli/partition_test/multiple_partitions.bats

```diff
@@ -0,0 +1,121 @@
+    export STREAMING_PROCESSING_UNIT_TWO_PUB_ADDR="127.0.0.1:9216"
+    export STREAMING_PROCESSING_UNIT_TWO_PRV_ADDR="127.0.0.1:9217"
+    export STREAMING_PROCESSING_UNIT_ONE_ID=1016
+    export STREAMING_PROCESSING_UNIT_TWO_ID=2016
+
+    PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME=$(random_string)
+    export PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME
+
+    PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME=$(random_string)
+    export PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME
+    debug_msg "Topic name: $PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME"
+
+    echo "Finishing Setup Process"
+    sleep 10
+}
+
+teardown_file() {
+    echo "Tearing down, shutting down cluster components"
+    "$FLUVIO_BIN" topic delete "$PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME"
+    fluvio cluster shutdown --local
+}
+
+@test "Setup environment" {
+    echo "Creating profile \"$FLUVIO_PROFILE_LOCAL\""
+    "$FLUVIO_BIN" profile add "$FLUVIO_PROFILE_LOCAL" "$STREAMING_CONTROLLER_PUB_ADDR"
+
+    echo "Executing Streaming Controller"
+    "$FLUVIO_BIN" run sc --local \
+        --bind-public $STREAMING_CONTROLLER_PUB_ADDR \
+        --bind-private $STREAMING_CONTROLLER_PRV_ADDR & disown
+    assert_success
+    sleep 5
+
+    echo "Register SPU 1"
+    "$FLUVIO_BIN" cluster spu register \
+        --id $STREAMING_PROCESSING_UNIT_ONE_ID \
+        --public-server "$STREAMING_PROCESSING_UNIT_ONE_PUB_ADDR" \
+        --private-server "$STREAMING_PROCESSING_UNIT_ONE_PRV_ADDR"
+    assert_success
+
+    echo "Register SPU 2"
+    "$FLUVIO_BIN" cluster spu register \
+        --id $STREAMING_PROCESSING_UNIT_TWO_ID \
+        --public-server "$STREAMING_PROCESSING_UNIT_TWO_PUB_ADDR" \
+        --private-server "$STREAMING_PROCESSING_UNIT_TWO_PRV_ADDR"
+    assert_success
+
+    echo "Initialize SPU 1"
+    "$FLUVIO_BIN" run spu \
+        --sc-addr $STREAMING_CONTROLLER_PRV_ADDR \
+        --id $STREAMING_PROCESSING_UNIT_ONE_ID \
+        --public-server "$STREAMING_PROCESSING_UNIT_ONE_PUB_ADDR" \
+        --private-server "$STREAMING_PROCESSING_UNIT_ONE_PRV_ADDR" & disown
+    assert_success
+    sleep 5
+
+    echo "Initialize SPU 2"
+    "$FLUVIO_BIN" run spu \
+        --sc-addr $STREAMING_CONTROLLER_PRV_ADDR \
+        --id $STREAMING_PROCESSING_UNIT_TWO_ID \
+        --public-server "$STREAMING_PROCESSING_UNIT_TWO_PUB_ADDR" \
+        --private-server "$STREAMING_PROCESSING_UNIT_TWO_PRV_ADDR" & disown
+    assert_success
+    sleep 5
+}
+
+@test "Create a topic for P/C Multiple Partitions" {
+    echo "Creates Topic: $PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME for P/C Multiple Partitions"
+    run timeout 15s "$FLUVIO_BIN" topic create "$PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME" \
+        --partitions 2 \
+        --replication 2
+
+    assert_success
+}
+
+@test "Produces on topic for P/C Multiple Partitions" {
+    echo "Produces value \"1\" on $PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME"
```

Comment:
instead of hard coding, this should just take partition parameter so can be based on partitioning parameter